.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,139 @@
+---
+name: Bug Report
+description: Create a report to help us improve.
+labels: [bug]
+assignees: aio-libs/triagers
+body:
+- type: markdown
+  attributes:
+    value: |
+      **Thanks for taking a minute to file a bug report!**
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+      _Please fill out the form below with as many precise
+      details as possible._
+
+      [issue search]: ../search?q=is%3Aissue&type=issues
+
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >-
+      A clear and concise description of what the bug is.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: To Reproduce
+    description: >-
+      Describe the steps to reproduce this bug.
+    value: |
+      1. Have certain environment
+      2. Run given code snippet in a certain way
+      3. See some behavior described
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Expected behavior
+    description: >-
+      A clear and concise description of what you expected to happen.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Logs/tracebacks
+    description: >-
+      |
+      If applicable, add logs/tracebacks to help explain your problem.
+      Paste the output of the steps above, including the commands
+      themselves and their output/traceback etc.
+    render: Python traceback
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: Python Version
+    description: Attach your version of Python.
+    render: console
+    value: |
+      $ python --version
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: aiohttp Version
+    description: Attach your version of aiohttp.
+    render: console
+    value: |
+      $ python -m pip show aiohttp
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: multidict Version
+    description: Attach your version of multidict.
+    render: console
+    value: |
+      $ python -m pip show multidict
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: yarl Version
+    description: Attach your version of yarl.
+    render: console
+    value: |
+      $ python -m pip show yarl
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: OS
+    placeholder: >-
+      For example, Arch Linux, Windows, macOS, etc.
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Related component
+    description: >-
+      aiohttp is both server framework and client library.
+      For getting rid of confusing make sure to select
+      'server', 'client' or 'both'.
+    options:
+      - Server
```

Comment:
Dedent this list so the style is aligned with the rest of the sequences in this file.

Reply:
Yes, but YAML doesn't require it. All the items under the "body" key, for example, don't have an extra indent so I'd like this not to have one either. Basically, you need to remove two spaces.

Reply:
OK, I just assumed that the options list must be indented so that the form can be rendered correctly.